app/index.html

```diff
@@ -13,6 +13,20 @@
     <link rel="stylesheet" href="assets/mcac.css">
 
     {{content-for 'head-footer'}}
+
+    <script>
+  (function(i,s,o,g,r,a,m)
```

Comment:
A few things wrong here:

 - It's not indented properly. The script tag does not open and close at the same level of indentation.
 - The script has `type` property... set it to `"text/javascript"`
 - Move the whole script to the end of the body. As a best practice, you should only put JavaScript in the `head` when it's absolutely necessary. See: http://stackoverflow.com/a/2451441/567090